packages/smart-wallet/package.json

```diff
@@ -26,6 +27,7 @@
   "dependencies": {
     "@agoric/assert": "^0.6.1-u11wf.0",
     "@agoric/casting": "^0.4.3-u13.0",
+    "@agoric/deploy-script-support": "^0.10.4-u13.0",
```

Comment:
I wonder if that could be moved to `devDependencies`.
not sure it matters, much, though...

Reply:
Good question, which we can punt on for now.